src/pages/definePassengerType.tsx

```diff
@@ -58,8 +58,8 @@ export const getFieldsets = (errors: ErrorInfo[], passengerType: string): RadioC
                 value: 'Yes',
                 dataAriaControls: 'age-range-required-conditional',
                 label: 'Yes',
-                hint: {
-                    id: 'define-passenger-age-range-hint',
+                inputHint: {
+                    id: 'define-passenger-age-range-inputHint',
```

Comment:
I think this ID can stay as `define-passenger-age-range-hint`?

Reply:
had an issue with the tests so had to change but let me check again

Reply:
sorted it